connector/src/main/scala/com/basho/riak/spark/rdd/partitioner/RiakCoveragePlanBasedPartitioner.scala

```diff
@@ -128,8 +147,8 @@ object RiakCoveragePlanBasedPartitioner extends Logging {
         DumpUtils.dump(result, "\n") +
         "\n----------------------------------------\n")
 
-      throw new IllegalStateException("Some coverage entries do not belongs to the spark " +
-        "partitions, it wil cause incomplete data load, see log for more datails.")
+      throw new IllegalStateException("Some coverage entries do not belong to any of the created spark " +
+        "partition. This will lead to incomplete data load, see log for more datails.")
```

Comment:
"partition. This will lead to incomplete data load, see log for more datails."
->
"partitions. This will lead to an incomplete data load, see log for more details."